RELEASE/scripts/autoscend/iotms/mr2017.ash

```diff
@@ -1765,8 +1765,8 @@ boolean makeGenieWish(effect eff)
 
 boolean canGenieCombat()
 {
-	if(item_amount($item[Genie Bottle]) == 0 && item_amount($item[pocket wish]) == 0)
-	{
+	if(item_amount($item[genie bottle]) == 0 && item_amount($item[pocket wish]) == 0 && !auto_is_valid($item[genie bottle]))
+	{ // can't use genie bottle in BHY
```

Comment:
again. are you sure?
also this comment is really redundant with auto_is_valid which is by and large self documenting